components/automate-ui/src/app/pages/policy/details/policy-details.component.html

```diff
@@ -22,59 +22,57 @@
           </tr>
         </tbody>
       </table>
-        <chef-tab-selector (change)="onSelectedTab($event)" [value]="tabValue">
-          <chef-option value='definition'>Definition</chef-option>
-          <chef-option value='members' routerLink="./" fragment="members">Members</chef-option>
-        </chef-tab-selector>
-      </chef-page-header>
-    <section class="page-body">
-      <ng-container *ngIf="tabValue === 'definition'">
-        <chef-toolbar>
-          <chef-clipboard value={{policyJSON}} label="Copy Definition" icon=""></chef-clipboard>
-        </chef-toolbar>
-        <section id="json-bg">
-          <chef-snippet [code]="policy | json" lang="json"></chef-snippet>
-        </section>
-      </ng-container>
-      <ng-container *ngIf="tabValue === 'members'">
-        <div *ngIf="(members$ | async)?.length === 0" class="empty-state">
-          <p>Add some members to get started!</p>
+      <chef-tab-selector (change)="onSelectedTab($event)" [value]="tabValue">
+        <chef-option value='definition'>Definition</chef-option>
+        <chef-option value='members'>Members</chef-option>
+      </chef-tab-selector>
+    </chef-page-header>
+    <section class="page-body" *ngIf="tabValue === 'definition'">
+      <chef-toolbar>
+        <chef-clipboard value={{policyJSON}} label="Copy Definition" icon=""></chef-clipboard>
+      </chef-toolbar>
+      <section id="json-bg">
+        <chef-snippet [code]="policy | json" lang="json"></chef-snippet>
+      </section>
+    </section>
+    <section class="page-body" *ngIf="tabValue === 'members'">
+      <div *ngIf="(members$ | async)?.length === 0" class="empty-state">
```

Comment:
❓ does this _need_ the `?`?

Reply:
It does. Anytime we want to deference a property on an observable brought into the template via an async pipe, that needs to be there because the object will be undefined for some indeterminate period before it resolves.

Tried to find a good reference for async pipes that covers it, but only found a couple that sort of explain it:
https://blog.angular-university.io/angular-reactive-templates/
https://ultimatecourses.com/blog/angular-ngif-async-pipe